geomesa-process/README.md

```diff
@@ -3,15 +3,27 @@
 The following analytic processes are available and optimized on GeoMesa data stores.
 
 * `DensityProcess` - computes a density heatmap for a CQL query
+*  HashAttributeColorProcess and HashAttributeProcess - computes an additional 'hash' attribute which is useful for styling.
+* `KNearestNeighborSearchProcess` - performs a KNN search
+*  Point2PointProcess - aggregates a collection of points into a collection of line segments
+* `ProximitySearchProcess` - performs a nearest neighbor search
+*  SamplingProcess - uses statistical sampling to reduces the features returned by a query
 * `StatsIteratorProcess` - returns various stats for a CQL query
 * `TubeSelectProcess` - performs a correlated search across time/space dimensions
-* `ProximitySearchProcess` - performs a nearest neighbor search
 * `QueryProcess` - optimizes GeoMesa queries in WPS chains
-* `KNearestNeighborSearchProcess` - performs a KNN search
 * `UniqueProcess` - identifies unique values for an attribute in results of a CQL query
 
+
 ## Installation
 
-To install the GeoMesa processes, first deploy the main GeoMesa GeoServer plugin in 
- `${GEOSERVER_HOME}/WEB-INF/lib`.  Then copy the `geomesa-process-accumulo1.5-${VERSION}.jar` into the
- same directory.
\ No newline at end of file
+The above extensions are particular to the Accumulo datastore.
+
+While they can be used independently, the common use case is to use them with GeoServer.  To deploy them in GeoServer, one will require a) the GeoMesa Accumulo datastore plugin, b) the GeoServer WPS extension zip [], and c) the geomesa-process-${VERSION}.jar to be deployed in ${GEOSERVER_HOME}/WEB-INF/lib.  
```

Comment:
[picky] I would put the JAR and file path in backticks.

Also, I am not sure what the empty square brackets in part B are for. Missing link?